### PR TITLE
BH eu2018: Gemalto Mobile Protector

### PR DESCRIPTION
**Pull request by enovella**
Done one obfuscator from https://github.com/rednaga/APKiD/issues/121

### CODE REVIEW

File: apkid/rules/apk/obfuscators.yara [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+{
+  meta:
+    description = "Gemalto"
+    url         = "https://www.gemalto.com"
+    author      = "Eduardo Novella"
+    sample      = "294f95298189080a25b20ef28295d60ecde27ee12361f93ad2f024fdcb5bdb0b"
+
+
+  strings:
+    $l1 = "lib/arm64-v8a/libmedl.so"
```
Comment (CalebFenton) [inline apkid/rules/apk/obfuscators.yara]: How about replace this with a regex to future proof against any new architectures?

File: apkid/rules/dex/obfuscators.yara [hunk 2]
```diff
@@ -241,3 +241,22 @@ rule aamo_str_enc : obfuscator
...
+{
+  meta:
+    description = "Gemalto"
+    url         = "https://www.gemalto.com/brochures-site/download-site/Documents/eba_ezio_on_mobile.pdf"
+    author      = "Eduardo Novella"
+    sample      = "294f95298189080a25b20ef28295d60ecde27ee12361f93ad2f024fdcb5bdb0b"
+
+
+  strings:
+    $p1 = "Lcom/gemalto/idp/mobile/"
```
Comment (CalebFenton) [inline apkid/rules/dex/obfuscators.yara]: Possible to build rule off of actual obfuscation? Perhaps some strings, decoder opcodes, etc?

File: apkid/rules/apk/obfuscators.yara (hunk 1 above)
Comment (enovella) [reply apkid/rules/apk/obfuscators.yara]: Perhaps it would be slower, right?

File: apkid/rules/dex/obfuscators.yara (hunk 2 above)
Comment (enovella) [reply apkid/rules/dex/obfuscators.yara]: These three packages are good enough to detect the different Gemalto SDKs. Of course, we can add more strings, but maybe we would miss some matches.

**Pull request merged**